packages/core/.eslintrc.js

```diff
@@ -1,22 +1,37 @@
 module.exports = {
   root: true,
-  parser: 'babel-eslint',
-  parserOptions: {
-    ecmaVersion: 2018,
-    sourceType: 'module',
-    ecmaFeatures: {
-      legacyDecorators: true
-    }
-  },
-  plugins: ['ember'],
-  extends: ['eslint:recommended', 'plugin:ember/recommended', 'prettier'],
+  parser: '@typescript-eslint/parser',
+  plugins: ['ember', 'prettier', 'qunit', '@typescript-eslint'],
+  extends: [
+    'eslint:recommended',
+    'plugin:ember/recommended',
+    'plugin:qunit/recommended',
+    'plugin:@typescript-eslint/recommended',
+    'prettier',
+    'prettier/@typescript-eslint'
+  ],
   env: {
     browser: true
   },
   rules: {
     'ember/no-jquery': 'warn',
     'ember/no-new-mixins': 'warn',
-    'ember/no-observers': 'warn'
+    'ember/no-observers': 'warn',
+
+    // cleanliness & consistency
+    'prefer-const': 'off', // const has misleading safety implications
```

Comment:
Do we want to eventually turn this on?

Reply:
We could, trying to keep the changes as small as possible without turning too much off